internal/dump/tar.go

```diff
@@ -38,6 +38,14 @@ const (
 	cISVTX = 0o1000 // Save text (sticky bit)
 )
 
+// substitute a uid or gid of -1 (which was converted to 2^32 - 1) with zero
+func tarIdentifier(id uint32) int {
+	if int32(id) == -1 {
```

Comment:
That will also replace the `-1` for 64-bit builds which don't require that workaround. Wouldn't it be better to only modify the identifiers if it is absolutely necessary, that is if `int(id) == -1`?

Reply:
I am fine with this less intrusive solution. I add a commit.